fabric-structures-v1/src/main/java/net/fabricmc/fabric/api/structures/v1/FabricStructureHelper.java

```diff
@@ -39,7 +40,7 @@ public static void setGenerationStep(FabricStructure structure) {
 	}
 
 	/**
-	 * Registers a structure in the structure registry, the serialization hashmap, and generation step hashmap
+	 * Registers a structure in the structure registry, the serialization hashmap, and generation step hashmap.
```

Comment:
Hashmap is impl detail, should be closer to (don't quote it literally):

```suggestion
	 * Registers a structure in the structure registry, it's generation step and serializer.
```

Reply:
Should be better in 71b24d8